.changeset/breezy-scissors-approve.md

```diff
@@ -2,4 +2,4 @@
 "@rhds/elements": patch
 ---
 
-`rh-tag`: outline variant should now get the correct background styles added.
+`<rh-tag>`: outline variant should now get the correct background styles added.
```

Comment:
@hellogreg @coreyvickery is this a good release notes line item for Bellsprout? Needs any revisions? Would you be happy automatically pulling this in to the next round of release notes?

Reply:
@bennypowers Next round is fine.